libr/core/panels.c

```diff
@@ -2261,7 +2261,11 @@ static void printSnow(RPanels *panels) {
 			continue;
 		}
 		if (r_cons_canvas_gotoxy (panels->can, snow->x, snow->y)) {
-			r_cons_canvas_write (panels->can, "*");
+			if (panels->fun == PANEL_FUN_SAKURA) {
+				r_cons_canvas_write (panels->can, Color_BMAGENTA"*"Color_RESET);
```

Comment:
leafs are not asterisks

Reply:
lmao right. but in Japan, we only focus on beautiful SAKURA flower falls and dismiss leaf anyway :P